doc/s390-upstream-prerequisites.md

```diff
@@ -0,0 +1,32 @@
+### s390 backporting
+
+**Prerequisite gcc patches (all backported to releases/gcc-11 branch):**
+- gcc-mirror/gcc@a1c1b7a IBM Z: Define NO_PROFILE_COUNTERS
+- gcc-mirror/gcc@0990d93 IBM Z: Use @PLT symbols for local functions in 64-bit mode
+- gcc-mirror/gcc@935b522 S/390: New option -mpic-data-is-text-relative
+- gcc-mirror/gcc@8753b13 IBM Z: fix section type conflict with -mindirect-branch-table
+
+**Prerequisite kernel patches:**
+- 69505e3d9a39 bug: Use normal relative pointers in 'struct bug_entry'
```

Comment:
Most likely this will end up in 5.19, so it's probably safe to add a v5.19 header for this one.

Reply:
Added